guides/docs/3-contribute.md

```diff
@@ -60,10 +60,10 @@ wait to hear from you.
 
 ## Behaving yourself
 
-[Code of Conduct](./CODE_OF_CONDUCT.md)
+[Code of Conduct](https://github.com/rangle/augury/blob/dev/CODE_OF_CONDUCT.md)
 
 In the interest of making the Augury project a safe and friendly
-place for people from diverse backgrounds, we'll be adhering to a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). As a contributor, you'll be expected to
+place for people from diverse backgrounds, we'll be adhering to a [Contributor Code of Conduct](https://github.com/rangle/augury/blob/dev/CODE_OF_CONDUCT.md). As a contributor, you'll be expected to
```

Comment:
For future reference, we could make the `CODE_OF_CONDUCT.md` a page on the site. Not sure if that's the where we want it, but it is a possibility. This would be outside the scope of this PR.